packages/fleather/lib/src/rendering/embed_proxy.dart

```diff
@@ -50,3 +50,114 @@ class RenderEmbedProxy extends RenderProxyBox implements RenderContentProxyBox {
   @override
   double get preferredLineHeight => size.height;
 }
+
+/// Proxy to an arbitrary embeddable [RenderBox].
+///
+/// Computes necessary editing metrics based on the dimensions of the childlren
+/// render boxes.
+class RenderGroupEmbedProxy extends RenderFlex
+    implements RenderContentProxyBox {
+  RenderGroupEmbedProxy({
+    TextDirection? textDirection,
+  }) : super(textDirection: textDirection);
+
+  @override
+  List<TextBox> getBoxesForSelection(TextSelection selection) {
+    if (selection.isCollapsed) {
+      return [];
+    }
+
+    final childIter = _childrenIterator();
```

Comment:
Shouldn’t we call each child getBoxesForSelection here?

or can a child have a specific implementation for it?